packages/client/tests/e2e/default-version/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
this one was downgraded from 6.1 somehow? all others were on 5.4, and I ran `pnpm install` with latest 8.6.7